net/paho.mqtt.c/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        eclipse paho.mqtt.c 1.3.9 v
+categories          net
+platforms           darwin
+maintainers         nomaintainer
+license             Eclipse Public License - v-2.0
```

Comment:
The value for the `license` field should be kept to specific values that MacPorts recognizes.  You can see the list of license values here: https://trac.macports.org/wiki/PortfileRecipes#licensekeyword

In this case, it looks like license should be set to `EPL-2`.